api/registrations/serializers.py

```diff
@@ -543,6 +543,8 @@ def create(self, validated_data):
                                              ' registered: {}'.format(', '.join(orphan_files_names)))
 
         try:
+            # Still validating metadata, but whether `registration_responses` or `registration_metadata` were populated
+            # on the draft, the other field was built and populated as well.  Both should exist.
             draft.validate_metadata(metadata=draft.registration_metadata, reviewer=reviewer, required_fields=True)
```

Comment:
We should consider validating `registration_responses` instead! (Leaving for now, to be addressed in https://openscience.atlassian.net/browse/ENG-366)

Reply:
ah, a lot of the archiving code still looking at `registration_metadata` though -